pkgs/development/python-modules/aiomysql/default.nix

```
@@ -28,7 +28,7 @@ buildPythonPackage rec {
 
   postPatch = ''
     substituteInPlace setup.py \
-      --replace "PyMySQL>=0.9,<=0.9.2" "PyMySQL"
+      --replace "PyMySQL>=0.9,<=0.9.3" "PyMySQL"
```

```suggestion
      --replace "PyMySQL>=0.9,<0.10.0" "PyMySQL"
```
Otherwise it breaks on the next version bump again.

---

Unless I'm misunderstanding how `substituteInPlace` works, this won't work. `"PyMySQL>=0.9,<=0.9.3"` is currently in the file and needs to be removed. This change will just cause it to fail to match.

---

🤦🏼 You are right. Sorry.